src/App.tsx

```diff
@@ -1,61 +1,33 @@
+import { useState } from 'react';
 import './App.scss';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoForm } from './components/TodoForm';
+import { TodoList } from './components/TodoList';
+
+export interface Todo {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+}
+
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+}
 
 export const App = () => {
+  const [todoList, setTodoList] = useState(todosFromServer);
```

Comment:
If `todosFromServer` is fetched asynchronously, setting it directly as the initial state might cause issues. Consider initializing `todoList` with an empty array and updating it once the data is fetched.